pisa/stages/data/toy_event_generator.py

```diff
@@ -48,8 +48,8 @@ def __init__(self,
         input_apply_keys = ('initial_weights',
                            'weights',
                            'weighted_aeff',
-                           'nominal_nu_flux',
-                           'nominal_nubar_flux',
+                           'nu_flux_nominal',
+                           'nubar_flux_nominal',
```

Comment:
Why did you change these names? (This will break code in the fridge if you do so...)

Reply:
(I count 38 files with "nominal_nu_flux" in them in the fridge, and only 2 with "nu_flux_nominal"; I am, however, seeing the flux stages in PISA using "nu_flux_nominal" naming convention ... not sure what the story is, but it might be a concern)

Reply:
I changed these names because they were changed in the flux stage of PISA when it was re-named to pi_barr_simple. And yes, it was annoying because it did break my code initially. I'm happy to revert this if we also change back the flux stage.

Reply:
I confirmed that this change happened in the commit https://github.com/IceCubeOpenSource/pisa/commit/a5e66450fc56494689ce858dbff20dc0bbc19765

Reply:
@ts4051 can you justify why you changed the names we're talking about above? Can you fix the code in fridge that change broke, or (what I would probably prefer at this point, but I am open to other opinions...) change back to the original naming scheme and fix whatever that change breaks?

Reply:
Since this re-naming change is (unfortunately) already in some parts of master, gonna merge this PR now